python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py

```diff
@@ -273,194 +324,238 @@
-betaHL = result.getHasoferReliabilityIndex()
-print("Hasofer index : ", betaHL)
-
+# We can get the Hasofer index with the `getHasoferReliabilityIndex` method which is the distance of
+# the design point to the origin:
+beta_HL = result.getHasoferReliabilityIndex()
+print("Hasofer index : ", beta_HL)
 
 # %%
-# We visualize it on the previous graph.
-cloud = ot.Cloud([designPointStandardSpace[0]], [designPointStandardSpace[1]])
+# We visualize the design point on the previous graph.
+cloud = ot.Cloud([design_point_standard_space])
 cloud.setColor("red")
 cloud.setPointStyle("fcircle")
-cloud.setLegend("design point")
-graphStandardSpace.add(cloud)
-graphStandardSpace.setGrid(True)
-graphStandardSpace.setLegendPosition("lower right")
+cloud.setLegend(r"design point $z^*$")
+graph_standard_space.add(cloud)
+graph_standard_space.setGrid(True)
+graph_standard_space.setLegendPosition("lower right")
 cc = ot.Curve(
-    [0.0, designPointStandardSpace[0]],
-    [0.0, designPointStandardSpace[1]],
+    [0.0, design_point_standard_space[0]],
+    [0.0, design_point_standard_space[1]],
     r"$\beta_{HL}$ distance",
 )
 cc.setLineStyle("dashed")
 cc.setColor("black")
-graphStandardSpace.add(cc)
-view = otv.View(graphStandardSpace)
+graph_standard_space.add(cc)
+graph_standard_space.setLegendPosition('topright')
+view = otv.View(graph_standard_space, square_axes=True)
 
 # %%
-# Estimating the failure probability integral
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The FORM approximation
+# ^^^^^^^^^^^^^^^^^^^^^^
 #
-# The last step of the FORM algorithm is to replace the domain of integration by the half-space at
-# the design point. In this simple example the half-space is delimited by the tangent at the design
-# point in the standard space.
+# The last step of the FORM algorithm is to replace the failure domain boundary by the hyperplane
+# which is tangent to the failure domain at the design point in the standard space.
+# To draw this hyperplane :math:`\mathcal{P}_{\vect{z}^*}`, we define the function from
+# :math:`\Rset^2` to :math:`\Rset` defined by:
 #
-# The expression of the failure domain boundary in the standard space is the composition of the
-# hyperbole :math:`h:x \mapsto 10/x` and the inverse transform on the first variable. We can compute
-# the gradient (here the first derivative of a 1D function :math:`h(u_0)` ) at any given point with the
-# getGradient method :
-
-# %%
-u0 = [designPointStandardSpace[0]]
-du0 = failureBoundaryStandardSpace.getGradient().gradient(u0)
-print("abscissa of the design point u0  = ", u0[0])
-print("value of the failure boundary at u0 = ", failureBoundaryStandardSpace(u0)[0])
-print("value of the gradient of the failure boundary at u0 = ", du0[0, 0])
-
-
-# %%
-# In the standard space the equation of the tangent :math:`\Pi_{u_0}(x)` is given by
+# .. math::
+#
+#    M \rightarrow \scalarproduct{\nabla \widetilde{\model}(\vect{z}^*)}{\vect{Z^*M}}
```

Comment:
I do not understand this equation:

![image](https://github.com/user-attachments/assets/d5ae5739-6adf-4450-9259-63ebda743b9f)

What is $\boldsymbol{Z^* M}$? Is $\langle \cdot \rangle$ the scalar product? Why is this involved in this equation?